proto/controlclient.proto

```diff
@@ -86,14 +86,26 @@ message _FlushCacheResponse {
 
 }
 
-message _GenerateTokenRequest {
-  // how many seconds do you want the api token to be valid for?
-  uint32 valid_for_seconds = 1;
+message _GenerateApiTokenRequest {
+  // generate a token that will never expire
+  message Never {}
+  // generate a token that has an expiry
+  message Expires {
+    // how many seconds do you want the api token to be valid for?
+    uint32 valid_for_seconds = 1;
+  }
+
+  oneof expiry {
+    Never never = 2;
+    Expires expires = 3;
+  }
+
+  reserved 1;
 }
 
-message _GenerateTokenResponse {
+message _GenerateApiTokenResponse {
   // the token that will be used to make requests against Momento backend
```

Comment:
What do you think about updating this slightly to say like "the api key used for authentication against Momento backend" or something, to try to call out that it's just the bit used for authentication?

Reply:
fixed in https://github.com/momentohq/client-protos/pull/127/commits/56d96792c8d8a421a528caf9168c801351a60e7f